modules-packages-crates/includes/1-scoping-terminology.md

```diff
@@ -0,0 +1,101 @@
+# Scoping Terminology
```

Comment:
Could the title of this unit be changed? Not sure if scoping terminology is descriptive enough. Some suggestions:
- Understand concepts behind code organization
- Understand how Rust programs are organizaed
- Learn more about packags, crates, and modules